version.sbt

```diff
@@ -0,0 +1 @@
+version in ThisBuild := "0.4.0-SNAPSHOT"
```

Comment:
You might be surprised but I actually never thought of the interaction between sbt-release and sbt-dynver (shows you how much I use sbt-release). I'm happy to consider changes in sbt-dynver for sbt-release and, actually, I'm also happy to collaborate to make sbt-release more "sbt-dynver ready".